common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
`backgrounds.json`, `content.json`, `faq.json`, `gear.json`, `loadingScreenTips.json`, `maintenance.json`, `merch.json`, `messages.json`, `overview.json`, `questsContent.json`, and `spells.json` are also indented with only 2 spaces.

The rest of the *.json files are indented with 4 spaces.